_includes/_home/_intro.html

```diff
@@ -0,0 +1,22 @@
+<section class="section section--intro">
+    <div class="section__inner">
+        <h1 class="section__callout" markdown="1">
```

Comment:
Woah, I didn't know about the `markdown="1"` feature. Very cool

Reply:
It's super useful!